pageserver/src/tenant/storage_layer/layer_desc.rs

```diff
@@ -0,0 +1,105 @@
+use std::ops::Range;
+use utils::{
+    id::{TenantId, TimelineId},
+    lsn::Lsn,
+};
+
+use crate::repository::Key;
+
+use super::{DeltaFileName, ImageFileName, LayerFileName};
+
+/// A unique identifier of a persistent layer. This is different from `LayerDescriptor`, which is only used in the
```

Comment:
We could remove `LayerDescriptor` and steal the name here. It's a little implied that it's `Persistent` because of the association with "file descriptor" (as long as it's documented in the docstring)

Reply:
Sounds good but I think you have a pending PR for moving LayerDescriptor. We can do this after all of our PRs are merged to avoid conflicts.

Reply:
Just closed it.